src/precice/impl/SolverInterfaceImpl.cpp

```diff
@@ -804,7 +804,7 @@ void SolverInterfaceImpl::getMeshVertexIDsFromPositions(
   }
 }
 
-int SolverInterfaceImpl::setMeshEdge(
+void SolverInterfaceImpl::setMeshEdge(
     MeshID meshID,
     int    firstVertexID,
     int    secondVertexID)
```

Comment:
Now that you are touching these API functions, I was wondering whether it is a good time to switch from an `int forstVertexID` to ` unsigned int` or maybe even `VertexID` (of type size_type) or something. I guess it will be  inevitable in the long run when going towards larger problems.

Reply:
Upvote, but I think it could be easier in a separate PR. Is there already a v3.0 issue for this?

Reply:
Very good point indeed

Reply:
> Is there already a v3.0 issue for this?

Regarding 64-bit ids: #887

We can change the parameter types (and names) now. This doesn't change the API as is.
Then we change the types in 3.0.0.

Reply:
To get this PR done soonish, I'll touch the interface types later.